.github/workflows/validate-pr.yml

```diff
@@ -0,0 +1,57 @@
+# Validates the format of a PR title and body. Because the title will become the default commit message on squash-merge,
+# it should follow the conventional commit format and be 70 characters long or fewer. Because the body will later be
```

Comment:
70 characters is too much if a PR doesn't require more than a sentence for its summary

Reply:
Also what's the way to bypass this? Will all the tiny "chore" PRs require this section?

Reply:
The audience of the header is a customer reading release notes and the purpose of the validation is to ensure we pause to communicate with a customer in mind. I think 70 characters is a pretty reasonable minimum. In our CHANGELOG.md history, I found just one example of an entry that's shorter (60) and it included no context for the change.

For chores, it might be possible to merge the PR with the check failing (it was on my fork though maybe we have it locked down more in the base repo). Or we can add logic to ignore the description length if it's a chore. That's an easy change to make so I'll just push that up.